apps/meteor/ee/server/models/raw/LivechatRooms.ts

```diff
@@ -726,4 +733,33 @@ export class LivechatRoomsRawEE extends LivechatRoomsRaw implements ILivechatRoo
 			...extraQuery,
 		});
 	}
+
+	updateMergedContactIds(
+		contactIdsThatWereMerged: ILivechatContact['_id'][],
+		newContactId: ILivechatContact['_id'],
+		options?: UpdateOptions,
+	): Promise<UpdateResult | Document> {
+		return this.updateMany({ contactId: { $in: contactIdsThatWereMerged } }, { $set: { contactId: newContactId } }, options);
+	}
+
+	findClosedRoomsByContactAndSourcePaginated({
+		contactId,
+		source,
+		options = {},
+	}: {
+		contactId: string;
+		source?: string;
+		options?: FindOptions;
+	}): FindPaginated<FindCursor<IOmnichannelRoom>> {
+		return this.findPaginated<IOmnichannelRoom>(
+			{
+				contactId,
```

Comment:
looks like we'll need an index for `contactId` as well

Reply:
We'll fix this [here](https://rocketchat.atlassian.net/browse/SCI-176?atlOrigin=eyJpIjoiZWY5MTM0OGYzYjVmNGM1Y2FiNGU2ZmYwZDU5NzI4M2IiLCJwIjoiaiJ9)